src/stretchy.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * This file provides support to buildMathML.js and buildHTML.js
+ * for stretchy wide elements rendered from background-image SVG files
+ * and other CSS trickery.
+ */
+
+const buildCommon = require("./buildCommon");
+const mathMLTree = require("./mathMLTree");
+const utils = require("./utils");
+
+const stretchyCodePoint = {
+    widehat : "^",
+    widetilde : "~",
+    undertilde : "~",
+    overleftarrow : "\u2190",
+    underleftarrow : "\u2190",
+    xleftarrow : "\u2190",
+    overrightarrow: "\u2192",
+    underrightarrow: "\u2192",
+    xrightarrow : "\u2192",
+    overbracket : "\u23b4",
+    underbrace : "\u23b5",
+    overbrace : "\u23de",
+    underbracket : "\u23df",
+    overleftrightarrow : "\u2194",
+    underleftrightarrow : "\u2194",
+    xleftrightarrow : "\u2194",
+    Overrightarrow : "\u21d2",
+    xRightarrow : "\u21d2",
+    overleftharpoon : "\u21bc",
+    xleftharpoonup : "\u21bc",
+    overrightharpoon : "\u21c0",
+    xrightharpoonup : "\u21c0",
+    xLeftarrow : "\u21d0",
+    xLeftrightarrow : "\u21d4",
+    xhookleftarrow : "\u21a9",
+    xhookrightarrow : "\u21aa",
+    xmapsto : "\u21a6",
+    xrightharpoondown : "\u21c1",
+    xleftharpoondown : "\u21bd",
+    xrightleftharpoons : "\u21cc",
+    xleftrightharpoons : "\u21cb",
+    xtwoheadleftarrow : "\u219e",
+    xtwoheadrightarrow : "\u21a0",
+    xLongequal : "=",
+    xtofrom : "\u21c4",
+};
+
+const mathMLnode = function(label) {
+    const node = new mathMLTree.MathNode(
+        "mo", [new mathMLTree.TextNode(stretchyCodePoint[label.substr(1)])]);
+    node.setAttribute("stretchy", "true");
+    return node;
+};
+
+const katexImagesData = {
+                 // height, depth, fileName
+    overleftarrow : [0.334, 0, "leftarrow"],
+    underleftarrow : [0.334, 0, "leftarrow"],
+    xleftarrow : [0.261, 0.261, "xleftarrow"],
```

Comment:
I used http://glyphrstudio.com/online/ to convert the KaTeX Main and KaTeX AMS ttf font files to SVG format. I then used the `path d= ... ` geometry to create the SVG files used in this PR. The fonts conveniently extracted into SVG paths with a scale in which 1000 inside the SVG viewBox equaled 1 em in the document. 

0.334 is because the arrowhead (from KaTeX AMS) is 334 units tall inside the font. Similarly, the KaTeX Main arrowhead, used for extensible arrows, is 522 units tall inside the font, and I split that distance around the math axis.